managing_providers/cloud_providers/cic_providers.md

```diff
@@ -0,0 +1,16 @@
+---
+---
+
+# IBM Cloud Infrastructure Center (CIC) Providers
+
+## IBM Cloud Infrastructure Center (CIC)
+
+IBM Cloud Infrastructure Center (CIC) is an advanced infrastructure management (OpenStack compatible) software that includes on-premises deployments of IBM z/VM®- and KVM-based Linux® virtual machines on the IBM Z® and LinuxONE platforms.
+
+## Adding IBM CIC Providers
+
+{% include openstack/adding-a-cic-cloud-provider.md %}
+
+## Configuring the Overcloud to Store Events
+
+{% include openstack/configure-overcloud-to-store-events.md %}
```

Comment:
I'm not sure if this is necessary or not, best to leave it out for now we can check with the IBM CIC team if there is anything required for events